tools/wasm-sourcemap.py

```diff
@@ -230,6 +230,9 @@ def read_dwarf_entries(wasm, options):
     for file in re.finditer(r"file_names\[\s*(\d+)\]:\s+name: \"([^\"]*)\"\s+dir_index: (\d+)", line_chunk):
       dir = include_directories[file.group(3)]
       file_path = (dir + '/' if file.group(2)[0] != '/' else '') + file.group(2)
+      if file_path.startswith('/emsdk/emscripten/'):
+        sub_dir = os.path.relpath(file_path, '/emsdk/emscripten')
```

Comment:
I wonder if we can factor out this constant which is duplicated here and also shared with `system_libs.py`?

Reply:
Done